samza-api/src/main/java/org/apache/samza/system/StreamSpec.java

```diff
@@ -289,4 +292,8 @@ public static StreamSpec createCoordinatorStreamSpec(String physicalName, String
   public static StreamSpec createCheckpointStreamSpec(String physicalName, String systemName) {
     return new StreamSpec(CHECKPOINT_STREAM_ID, physicalName, systemName, 1);
   }
+
+  public static StreamSpec createStreamAppenderSpec(String physicalName, String systemName, int partitionCount) {
```

Comment:
nit: for consistency "createStreamAppenderStreamSpec"

Reply:
Good catch. My intention was your suggestion. I'll edit that.